spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -483,6 +493,14 @@ protected PartitioningWriter<PositionDelete<InternalRow>, DeleteWriteResult> new
         SparkFileWriterFactory writers,
```

Comment:
Shall we update the comment above this method to mention DVs?

Reply:
Done!